BlazorTemplater/ComponentRenderer.cs

```diff
@@ -38,6 +38,17 @@ public ComponentRenderer()
 
         #region Services
 
```

Comment:
I moved your new method into the `#services` region next to `AddService` as that makes sense

Reply:
thanks for the fix and advices!